source/includes/api/_metadata.md

```diff
@@ -1,240 +1,537 @@
+ 
+### Getting all metadata
+
+> Getting a list of all metadata
 
 ```shell
-curl -X POST https://api.resourcewatch.org/v1/dataset/<dataset-id>/metadata \
--H "Content-Type: application/json"  -d \
- '{
-   "application": <app>,
-   "language": <language>
-  }'
+curl -L -X GET 'https://api.resourcewatch.org/v1/metadata?application=rw,gfw&language=en,es&type=dataset&limit=1' \
+-H 'Content-Type: application/json' \
 ```
 
-> Create metadata for a widget
-
+> Getting a list of all metadata with optional parameters
 
 ```shell
-curl -X POST https://api.resourcewatch.org/v1/dataset/<dataset-id>/widget/<widget-id>/metadata \
--H "Content-Type: application/json"  -d \
- '{
-   "application": <app>,
-   "language": <language>
-  }'
+curl -L -X GET 'https://api.resourcewatch.org/v1/metadata?application=rw,gfw&language=en,es&type=dataset&limit=1' \
+-H 'Content-Type: application/json' \
+```
+
+> Example response
+
+```json
+{
+    "data": [
+        {
+            "id": "57cfffd6129c3c100054c383",
+            "type": "metadata",
+            "attributes": {
+                "dataset": "0ffe5ad6-c920-421b-b69c-bc8893155df4",
+                "application": "gfw",
+                "resource": {
+                    "id": "0ffe5ad6-c920-421b-b69c-bc8893155df4",
+                    "type": "dataset"
+                },
+                "language": "en",
+                "name": "",
+                "description": "",
+                "source": "",
+                "citation": "",
+                "license": "",
+                "info": {
+                    "organization": "",
+                    "license": "",
+                    "source": "",
+                    "citation": "",
+                    "description": "",
+                    "short-description": "",
+                    "subtitle": "",
+                    "title": ""
+                },
+                "createdAt": "2016-12-13T10:02:28.337Z",
+                "updatedAt": "2016-12-13T10:03:02.445Z",
+                "status": "published"
+            }
+        }
+    ]
+}
 ```
 
-> Create metadata for a layer
+This endpoint will allow you to get the list of the metadata available in the API, and it's the only endpoint that will allow you to load metadata for multiple resources at once.
 
+It's worth pointing out that, unlike other similar endpoints on the RW API, this endpoint does NOT have pagination by default. If you query it without any of the optional parameters, you will get a list of all metadata objects for all datasets, widgets and layers. This is strongly discouraged, and you should not rely on this behavior - when using this endpoint, you should aim to use a combination of parameters to narrow down your response pool. You should also expect a future update to this endpoint to introduce pagination on all responses, so try to keep this in mind when using this endpoint on your application.
```

Comment:
"A wild deprecation notice appears" :sweat_smile: I'd flag the last sentence in its own paragraph (or at the very least in bold) since it's something that must not be missed by users reading these docs!